.github/scripts/setup_spark_remote.sh

```diff
@@ -2,31 +2,36 @@
 
 set -xv
 
-mkdir -p $HOME/spark
-cd $HOME/spark
+mkdir -p "$HOME"/spark
+cd "$HOME"/spark || exit
 
 version=$(wget -O - https://dlcdn.apache.org/spark/ | grep 'href="spark' | grep -v 'preview' | sed 's:</a>:\n:g' | sed -n 's/.*>//p' | tr -d spark- | tr -d / | sort -r --version-sort | head -1)
```

Comment:
Add error checking after the version assignment. For, example,
if [ -z "$version" ]; then echo "Failed to extract Spark version"; exit 1; fi

Reply:
done